blog/2023-12-21-alert-configuration-manager/index.md

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
there is something acting strange with the slug/metadata section and I think it is this empty line on top
see an example of another blogpost
![image](https://github.com/netdata/blog/assets/82235632/97b61d04-c120-4543-839b-1e1a9b809038)

Reply:
I fixed it the `---` were not on the first line.. :D